### DISCUSSION TITLE
Setting added status options: $t:published...

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#10884] [Archive]
In the "Interface" option of the Status field, I see `$t:published`
Does $t stands for translation?
Anyhow, where can I set the translation of added Status options?

**Comment by azrikahar** [comment]
> Does $t stands for translation?

You're correct! `$t:published` uses the app translation (with contributions from our community via https://locales.directus.io/) which can be found here:

https://github.com/directus/directus/blob/8cdb6edfca1c217b02d8bc7965cd20116c30f075/app/src/lang/translations/en-US.yaml#L27

> Anyhow, where can I set the translation of added Status options?

Unfortunately that is currently not possible. However, this use case is an ongoing discussion over here: https://github.com/directus/directus/discussions/3075
